app/commands/remove_place.rb

```diff
@@ -0,0 +1,35 @@
+class RemovePlaceCommand < BaseCommand
+  def initialize(user, args)
+    @user = user
+
+    matches = args.match(RemovePlaceCommand.regex)
+    @name = matches[:name]
```

Comment:
It makes sense to me to pass in actual arguments instead of a just a string, but I also like keeping the messiness of each command's parsing out of the route. I could be convinced either way.